acceptance/bundle/templates/default-python/integration_classic/output.txt

```diff
@@ -182,20 +183,21 @@ Validation OK!
 -          "dev": "[USERNAME]"
          },
          "tasks": [
-@@ -72,5 +60,5 @@
+@@ -72,5 +61,5 @@
              "job_cluster_key": "job_cluster",
              "notebook_task": {
 -              "notebook_path": "/Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/dev/files/src/notebook"
 +              "notebook_path": "/Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/prod/files/src/notebook"
              },
              "task_key": "notebook_task"
-@@ -107,5 +95,4 @@
+@@ -107,5 +96,5 @@
          ],
          "trigger": {
 -          "pause_status": "PAUSED",
++          "pause_status": "UNPAUSED",
```

Comment:
I suspect this is the diff between dev and prod, indicating that the default was **not** sent for prod.

Can you confirm?

Reply:
This is indeed a diff between dev and prod output for "validate -o json" and the pause_status=UNPAUSED **is now** sent for prod but it **was not** before:

- (NEW) pause_status: UNPAUSED is present in prod configuration. It comes from defaults added in this PR. It matches what terraform would add anyway. (We'll be able to verify that better once RecordRequests for integration tests by @shreyas-goenka is landed https://github.com/databricks/cli/pull/2720)

- (UNCHANGED) pause_status: PAUSED is present in dev configuration. It comes from ApplyTargetMode.